packages/react-integration/cypress/integration/datalist.spec.ts

```diff
@@ -16,28 +16,19 @@ describe('Data List Demo Test', () => {
   });
 
   it('Verify rows selectable', () => {
-    cy.get('#row1.pf-m-selectable').should('be.visible');
-    cy.get('#row2.pf-m-selectable').should('be.visible');
+    cy.get('#row1.pf-m-selectable').should('exist');
+    cy.get('#row2.pf-m-selectable').should('exist');
 
-    cy.get('#row1.pf-m-selected').should('not.be.visible');
-    cy.get('#row2.pf-m-selected').should('not.be.visible');
+    cy.get('#row1.pf-m-selected').should('not.exist');
+    cy.get('#row2.pf-m-selected').should('not.exist');
 
     cy.get('#row1').click();
-    cy.get('#row1.pf-m-selected').should('be.visible');
-    cy.get('#row2.pf-m-selected').should('not.be.visible');
+    cy.get('#row1.pf-m-selected').should('exist');
+    cy.get('#row2.pf-m-selected').should('not.exist');
 
     cy.get('#row2').click();
-    cy.get('#row1.pf-m-selected').should('not.be.visible');
-    cy.get('#row2.pf-m-selected').should('be.visible');
-  });
-
-  it('Verify actions ', () => {
```

Comment:
Don't we want to keep this to test the interaction?

Reply:
We already test this interaction in Dropdown.